validator/client/beacon-api/json_rest_handler.go

```diff
@@ -43,13 +49,16 @@ func (c beaconApiJsonRestHandler) GetRestJsonResponse(apiEndpoint string, respon
 // PostRestJson sends a POST requests to apiEndpoint and decodes the response body as a JSON object into responseJson. If responseJson
 // is nil, nothing is decoded. If an HTTP error is returned, the body is decoded as a DefaultErrorJson JSON object instead and returned
 // as the first return value.
-func (c beaconApiJsonRestHandler) PostRestJson(apiEndpoint string, headers map[string]string, data *bytes.Buffer, responseJson interface{}) (*apimiddleware.DefaultErrorJson, error) {
+func (c beaconApiJsonRestHandler) PostRestJson(ctx context.Context, apiEndpoint string, headers map[string]string, data *bytes.Buffer, responseJson interface{}) (*apimiddleware.DefaultErrorJson, error) {
 	if data == nil {
 		return nil, errors.New("POST data is nil")
 	}
 
 	url := c.host + apiEndpoint
-	req, err := http.NewRequest("POST", url, data)
+	req, err := http.NewRequestWithContext(ctx, "POST", url, data)
```

Comment:
Can we use `http.MethodPost` here, like `GetRestJsonResponse` does?

Reply:
Done!